COM-2653:B2B P2 - Email templates for email and print quote

Rameshdesaiise opened the pull request:
Created quote summary template in back-office and emails

Commit:
COM-2653:B2B P2 - Email templates for email and print quote

 Created quote summary template in back-office and emails
 Updated email and backoffice topics

kibo-srikanthbandlamudi commented on labels/en-US.json:
```diff
@@ -534,12 +534,20 @@
   "quantityToReturn": "Quantity to Return",
   "quickOrder": "Quick Order",
+  "quoteAccountName":"Account Name",
+  "quoteAdjustments": "Adjustments",
```
I don't see a quoteAdjustments in de-DE.json

Commit:
COM3256: B2B P2 - Email & print Quote template issues

Removed the “Quote Name” text before the actual quote name in the title/header.

Add a Quote Number field after Account Name but before Quote Name. Map it to Number on the Quote.

Rename the label for Submitted Date to Created Date. Map it to AuditInfo.CreateDate on the Quote instead of SubmittedDate (which is getting removed).

Rename the label for Submitted By to Created By. This field should do a lookup of the UserId field on the Quote. This should always match the buyer's user ID which should be one of the user's on the account.
Add a colon to Shipping Method to match the other fields. Should say Shipping Method:.

Commit:
COM-3474: quote summary Template does not display shipping info

Replaced from destinations  to fulfillmentInfo contracts to get Address and shipping info for both print quote and email quote template.
Updated itemTotal,shippingInformation, handling and taxAndDuty.
Updated Template Name to Quote-summary.

Commit:
Fix whitespace
